vm/src/stdlib/symtable.rs

```diff
@@ -205,7 +205,7 @@ mod decl {
 
         #[pymethod(name = "is_nonlocal")]
         fn is_nonlocal(&self) -> bool {
-            matches!(self.symbol.scope, SymbolScope::Nonlocal)
+            matches!(self.symbol.scope, SymbolScope::Free)
```

Comment:
This is wrong, is_nonlocal should be a flag (or a boolean field, as we have it for now) on Symbol, but I figured symtable isn't used all that much so I didn't implement it, I'm sure it would be easy though.